index.html

```diff
@@ -7,7 +7,24 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div class="container"></div>
+    <div class="container">
+      <div class="container-list">
+        <h3 class="title pending">
+          대기중 (<span class="count count-pending">0</span>)
+        </h3>
+        <ul class="list list-pending">
+        </ul>
+        <h3 class="title done">
+          완료됨 (<span class="count count-done">0</span>)
+        </h3>
+        <ul class="list list-done">
+        </ul>
+      </div>
+      <form class="form container-form">
```

Comment:
아래 CSS에서 보면 `form`이란 css 클래스명이 사용되지 않고 `container-form`에서 `container`는 어떤 의미를 가지는지 명확하지 않게 느껴질 수 있을 것 같아요. 대신, 이 `form`은 이 페이지에서 하나밖에 없는, todo를 입력하는 form이니까, 클래스명 대신 id로 지정해보는 것은 어떨까요?

```suggestion
      <form id="todo-form">
```

Reply:
이를 감싸고 있는 container CSS속성에 flex로 배치되었기 때문에 클래스명도 비슷하게 줘야겠다는 생각이었어요!!
`form`클래스는 전혀 사용하지 않으니 없어도 되고, 하나밖에 없으니 id로 `todo-form`로 지정하는 것도 좋은 생각이네요.